CODEOWNERS

```diff
@@ -136,6 +136,8 @@
 
 /Workbooks/Online*Experimentation/** @microsoft/exp-devs
 
+/Workbooks/Azure*Monitor*-*Workspaces/Customer*Statsbeat*Insights/** @microsoft/TelReach-Devs
```

Comment:
currently this is rejected beacuse this github team does not have contributor access to this repo yet.  it *can*, someone just needs to follow the process in the contributing docs to send our team mail and we can add.

need to make sure that team is aware of this and will approve/maintain these things going forward